alf/algorithms/generator.py

```diff
@@ -363,6 +368,9 @@ def __init__(self,
                 ``fullrank_diag_weight``.  
             fullrank_diag_weight (float): weight on "extra" dimensions when 
                 forcing full rank Jacobian
+            block_inverse_mvp(bool): whether to use the more efficient block form
+                for inverse_mvp when ``functional_gradient`` is True. This
+                option only makes sense when ``noise_dim`` < ``output_dim``.
```

Comment:
this could be more clear: "this option is recommended only when ``noise_dim`` < ``output_dim``, as it is equivalent to the default behavior when ``noise_dim`` == ``output_dim``

Reply:
updated